### PR TITLE
Actually add the Testing project to the solution

### PR DESCRIPTION
**Pull request by ilhs1096069**
oops

### CODE REVIEW

**Review by roguefirework — APPROVED** [review]
It works, but at the cost of way to many pull requests

**Review by ilhs1041495 — COMMENTED** [review]
cangrejo

File: Core/WumpusCore/WumpusCore.sln [hunk 1]
```diff
@@ -5,7 +5,7 @@ VisualStudioVersion = 17.7.34018.315
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WumpusCore", "WumpusCore.csproj", "{20024C3F-3BAA-43EA-81CD-BD1137931B9B}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Testing", "..\Testing\Testing.csproj", "{5AF91DA1-E949-47DD-9242-B29C82C025FB}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WumpusTesting", "..\WumpusTesting\WumpusTesting.csproj", "{6F2E14B0-65E6-4C3C-8C13-D29D1715749E}"
```
Comment (1118422ILHS) [inline Core/WumpusCore/WumpusCore.sln:8]: are you kidding me?? 😭

Comment (ilhs1096069) [reply Core/WumpusCore/WumpusCore.sln:8]: cangrejo

**Review by 1118422ILHS — APPROVED** [review]
I said don't make silly mistakes again... Please in the future do basic tests on your branches before you attempt to merge into main. I want to be able to trust that you've done this, but that's becoming difficult. Please do the bare minimum

**Pull request merged**